api_tests/nodes/views/test_node_draft_registration_detail.py

```diff
@@ -433,6 +420,37 @@ def test_required_metaschema_questions_not_required_on_update(
         assert res.json['data']['attributes']['registration_metadata']['q2']['value'] == 'New response'
         assert 'q1' not in res.json['data']['attributes']['registration_metadata']
 
+    def test_required_registration_responses_questions_not_required_on_update(
```

Comment:
added a few more tests around this, plus there's model-level testing.  This will need a lot of QA.  Also, when we throw on validation when registrations are created, I think we're going to get a lot of test failures.

Reply:
I'm getting specific file things after turning on validation when registering a draft
```
*** ValidationError: {u'seen': True, u'user': u'b2u8h'} is not of type 'string'

Failed validating 'type' in schema['properties'][u'q7']['properties']['value']['properties'][u'uploader']['properties']['extra']['items']['properties']['data']['properties']['extra']['properties']['latestVersionSeen']:
    {'type': 'string'}

On instance[u'q7']['value'][u'uploader']['extra'][0]['data']['extra']['latestVersionSeen']:
    {u'seen': True, u'user': u'b2u8h'}
```

Reply:
Maybe updating OSF_UPLOAD_EXTRA_SCHEMA will help with ENG-366, also that schema might have to be updated to include the registration_responses things are being stashed there? just a thought, haven't dug much, but we do need to get validation turned on before we register!